py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/batcher.py

```diff
@@ -50,17 +53,21 @@ def batch_files(files: Iterable[(str, int)], threshold: int) -> list[Batch]:
     # iterate over file tuples, and add them to the ongoing batches. if a batch
     # is going to go past the threshold limit, move it to complete batches, and
     # create a new batch.
+    logging.info("Organizing files into batches.")
     for (filename, size) in files:
         try:
             config_type = ConfigType.from_filename(filename)
-        except:
-            # for now just print out the error.
-            print("Encontered Unknown File Type: %s" % filename)
+        except ConfigTypeFromFilenameException as config_exception:
+            logging.warning(config_exception)
+            continue
+        except Exception as exception:
+            loging.excption(exception)
```

Comment:
not sure if we want to re-raise here.

basically, if we encounter a filename we cant handle, do we want to:

A) log it and move on. it won't get picked up in a batch, and it will remain in the incoming s3 bucket until sentry yells at us and we fix it

OR

B) re-raise here and nothing gets batched until we fix the issue or remove the file.

I'm leaning A

Reply:
Agree that A is the way to go. Seems like we would be able to make a dashboard on splunk and watch for this type of thing.